lemur/sources/service.py

```diff
@@ -292,7 +294,10 @@ def delete(source_id):
 
     :param source_id: Lemur assigned ID
     """
-    database.delete(get(source_id))
+    source = get(source_id)
+    if source:
+        log_service.audit_log("delete_source", source.label, "Deleting source")
+    database.delete(source)
```

Comment:
should this also be part of `if source:`?

Reply:
Good call, I lost track of this with some other changes. I have confirmed that the original code did not throw any error is input to delete was `None`, thus moving delete inside if should be safe. It will not silence any past error.